### DISCUSSION TITLE
How do I update Vlad's diffusion? How does git pull work in VLAD diffusion? Can someone give me a guide and steps to follow?

### DISCUSSION THREAD

**Discussion by superprohero** [vladmandic/sdnext#1386] [Q&A] [answered]
help ༼ つ ◕_◕ ༽つ

**Comment by vladmandic** [comment]
https://github.com/vladmandic/automatic/discussions/1011#discussioncomment-5923750

**Reply by Tillerz** [reply]
make a new file in the automatic/ folder, name it upgrade.cmd

Type in:

```
@echo off
webui.bat --upgrade
```

And if you want to upgrade + start the ui, click this file instead. :)

**Reply by superprohero** [reply]
please tell me if I am doing this right.

![kjhkujhkhlkj](https://github.com/vladmandic/automatic/assets/133063465/f0781013-d17b-49c1-ba2a-210256669dcd)

**Reply by Tillerz** [reply, marked as answer]
yes, that should work.

**Reply by superprohero** [reply]
thanks man

**Reply by superprohero** [reply]
> yes, that should work.

what do I do now help?
![okdfjk;kf;lkw;](https://github.com/vladmandic/automatic/assets/133063465/9f67845b-0aa4-4311-b692-66b94835c857)

**Reply by superprohero** [reply]
> [#1011 (comment)](https://github.com/vladmandic/automatic/discussions/1011#discussioncomment-5923750)

help, what do I do now?

![okdfjk;kf;lkw;](https://github.com/vladmandic/automatic/assets/133063465/3dfc263b-89ba-4816-9c13-1128c501ac6a)

**Reply by vladmandic** [reply]
first, never ever post screenshots of logs - use copy&paste.
second, there had to be an error much earlier than part you've screenshotted, what you're showing is just a final (really bad) result.

**Reply by superprohero** [reply]
> first, never ever post screenshots of logs - use copy&paste. second, there had to be an error much earlier than part you've screenshotted, what you're showing is just a final (really bad) result.

I fixed it by reinstalling.

**Reply by leonnn1** [reply]
Dude why you treat people like that... cant you be nice?
